pkg/runcfanotify/runcfanotify.go

```diff
@@ -193,8 +193,21 @@ func NewRuncNotifier(callback RuncNotifyFunc) (*RuncNotifier, error) {
 	}
 
 	if !runcMonitored {
-		runcBinaryNotify.File.Close()
-		return nil, errors.New("no runc instance can be monitored with fanotify")
+		runcPath := os.Getenv("RUNC_PATH")
```

Comment:
I think it should be the same environment variable name as above (`$RUNTIME_PATH`) because `runcPaths` also includes `crun`.